libpod/image/image.go

```diff
@@ -329,16 +439,8 @@ func (ir *Runtime) LoadAllImagesFromDockerArchive(ctx context.Context, fileName
 		return nil, err
 	}
 
-	newImages := make([]*Image, 0, len(imageNames))
-	for _, name := range imageNames {
-		newImage, err := ir.NewFromLocal(name)
-		if err != nil {
-			return nil, errors.Wrapf(err, "error retrieving local image after pulling %s", name)
-		}
-		newImages = append(newImages, newImage)
-	}
```

Comment:
Nice. 

Absolutely non-blocking: If I’m checking correctly, doing the same to `LoadFromArchiveReference` would allow most of the `Load` functions involved in this PR to return `[]string`, and two of the callers of `Runtime.LoadImage` would eliminate a `strings.Split` (and the third one would now contain the only `strings.Join`).